code/game/objects/machinery/computer/station_alert.dm

```diff
@@ -103,4 +103,4 @@
 	else
 		icon_state = "atmos"
 	..()
```

Comment:
```suggestion
	return ..()
```

Reply:
process shouldnt return values unless its process kill

Reply:
should because some parent proc call might call it